src/Updater.php

```diff
@@ -20,6 +20,12 @@ class Updater implements UpdaterInterface {
 
   use StringTranslationTrait;
 
+  const CONFIG_NOT_FOUND = 0;
+  const CONFIG_ALREADY_APPLIED = 1;
+  const CONFIG_NOT_EXPECTED = 2;
```

Comment:
This name is a bit misleading. When I read it, I would expect it's the case when: `configuration is there, but it should not be`. I was thinking maybe `EXPECTED_CONFIG_NOT_COMPLIANT` or `UNEXPECTED_CONFIG`.

What do you think?